client/script.js

```diff
@@ -157,48 +153,6 @@ var orderComplete = function(clientSecret) {
   });
 };
 
-/* Display the OXXO details to your customer when the voucher has been created */
-var displayOxxoDetails = function(clientSecret) {
-  stripe.retrievePaymentIntent(clientSecret).then(function(result) {
-    var paymentIntent = result.paymentIntent;
-    const number = paymentIntent.next_action.display_oxxo_details.number;
-
-    var price = (paymentIntent.amount / 100).toFixed(2);
-    var numberFormat = new Intl.NumberFormat(["es-MX"], {
-      style: "currency",
-      currency: paymentIntent.currency,
-      currencyDisplay: "symbol"
-    });
-
-    document.querySelector(".order-amount").innerText = numberFormat.format(
-      price
-    );
-
-    document.querySelector(".oxxo-expiry-date").innerText = new Date(
-      paymentIntent.next_action.display_oxxo_details.expires_after * 1000
-    ).toLocaleDateString("es-MX");
-
-    document.querySelector(".oxxo-display").innerHTML = '<svg id="barcode"></svg>';
-
-    JsBarcode("#barcode", number, {
```

Comment:
Can you delete the script line that pulls in the JsBarcode library? We don't need it anymore

Reply:
Done